pom.xml

```diff
@@ -1508,18 +1439,37 @@
   </modules>
 
   <repositories>
-    <repository>
-      <id>kuali.nexus</id>
-      <name>Nexus Repository Manager</name>
-      <url>http://nexus.kuali.org/content/groups/public</url>
+      <repository>
+          <id>kuali.nexus</id>
+          <name>Nexus Repository Manager</name>
+          <url>http://nexus.kuali.org/content/groups/public</url>
+          <releases>
+              <enabled>true</enabled>
+          </releases>
+          <snapshots>
+              <enabled>false</enabled>
+          </snapshots>
+      </repository>
+      <repository>
+          <id>oss.sonatype.org</id>
+          <name>OSS Sonatype Staging</name>
+          <url>https://oss.sonatype.org/content/groups/staging</url>
+      </repository>
+  </repositories>
+
+  <pluginRepositories>
+    <pluginRepository>
+      <id>bintray</id>
+      <name>Groovy Bintray</name>
+      <url>https://dl.bintray.com/groovy/maven</url>
```

Comment:
This additional plugin repository was required for bringing in the latest versions of some Groovy plugins